using_images/xpaas_images/jws.adoc

```diff
@@ -9,7 +9,66 @@
 
 toc::[]
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+== Overview
+
+The Apache Tomcat 7 and Apache Tomcat 8 components of Red Hat JBoss Web Server 3 are available as containerized xPaaS images that are designed for use with OpenShift. Developers can use these images to quickly build, scale, and test Java web applications deployed across hybrid environments.
+
+[IMPORTANT]
+There are significant differences in the functionality in the JBoss Web Server xPaaS images compared to the regular release of JBoss Web Server.
+
+This topic details the differences between the JBoss Web Server xPaaS images and the regular release of JBoss Web Server, and provides instructions specific to running and configuring the JBoss Web Server xPaaS images. Documentation for other JBoss Web Server functionality not specific to the JBoss Web Server xPaaS images can be found in the https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Web_Server/[JBoss Web Server documentation on the Red Hat Customer Portal].
+
+The location of `_JWS_HOME_/tomcat<version>/` inside a JBoss Web Server xPaaS image is: *_/opt/webserver/_*.
+
+== Functionality Differences in the OpenShift JBoss Web Server xPaaS Images
+
+A major functionality difference compared to the regular release of JBoss Web Server is that there is no Apache HTTP Server in the OpenShift JBoss Web Server xPaaS images. All load balancing in OpenShift is handled by the OpenShift router, so there is no need for a load-balancing Apache HTTP Server with mod_cluster or mod_jk connectors. 
+
+== Installing the JBoss Web Server xPaaS Image Streams and Application Templates
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and Source-to-Image (S2I) application templates.
+
+. Clone the Application Template repository:
++
+----
+# git clone https://github.com/jboss-openshift/application-templates
+----
+. Create the image streams and S2I templates:
++
+----
+# oc create -f application-templates/jboss-image-streams.json -n openshift
+# oc create -f application-templates/webserver/ *.json -n openshift
+----
+
+You will now have the JBoss Web Server xPaaS images streams and S2I application templates available to use in your OpenShift project.
+
+== Using the JBoss Web Server xPaaS Image Source-to-Image (S2I) Process
+
+The recommended method to run and configure the OpenShift JBoss Web Server xPaaS images is to use the OpenShift S2I process together with the application template parameters and environment variables.
```

Comment:
We're saying "the recommended method.." but we're not offering an alternative. Would it be easier just to say:

"To run and configure the OpenShift JBoss Web Server xPaaS images, use the OpenShift S2I process with the application template parameters and environment variables."